src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-ok.html

```diff
@@ -0,0 +1,3 @@
+<p>
+    Every input step has a timeout especially for cases where there is no responce from the user..Jenkins aborts the job execution automatically.Param ok enables to question and agrees the action by Jenkins to be automated.
+</p>
```

Comment:
```suggestion
<div>
    Provides the text that will be displayed in the "ok" button that allows the Pipeline to continue.
</div>
```

I used the Pipeline snippet generator with these values:

![screencapture-mark-pc2-markwaite-net-8080-pipeline-syntax-2021-04-22-13_31_48-edit](https://user-images.githubusercontent.com/156685/115774770-3b81ed00-a36f-11eb-94a6-e03b191744c2.png)

to create this pipeline:

```
pipeline {
    agent any

    stages {
        stage('Hello') {
            steps {
                echo 'Hello World'
                script {
                def ret = input   id: 'A-custom-id-for-this-input-step',
                        message: 'This is the message',
                        ok: 'This is the OK button',
                        parameters: [
                            booleanParam(defaultValue: true,
                                         description: 'Boolean parameter to <code>input</code> step',
                                         name: 'A_BOOLEAN_PARAMETER_TO_INPUT')
                        ], 
                        submitter: 'mwaite,alice,bob,carl,devin,edgar',
                        submitterParameter: 'approvingSubmitterParameter'
                    echo "Has submitterParameter = ${ret.approvingSubmitterParameter}"
                    echo "Has A_BOOLEAN_PARAMETER_TO_INPUT = ${ret.A_BOOLEAN_PARAMETER_TO_INPUT}"
                }
                echo 'Goodbye World'
            }
        }
    }
}
```

that generated this result in the Jenkins Blue Ocean UI:

![screencapture-mark-pc2-markwaite-net-8080-blue-organizations-jenkins-pipeline-exploring-input-detail-pipeline-exploring-input-1-pipeline-2021-04-22-13_33_24](https://user-images.githubusercontent.com/156685/115775085-a3d0ce80-a36f-11eb-92b8-69ca7f9cd1eb.png)